tests/app/test_jinja_filters.py

```diff
@@ -455,3 +375,63 @@ def test_format_address_fields_with_uprn():
     assert (
         get_formatted_address(address_fields) == "7 Evelyn Street<br>Barry<br>CF63 4JG"
     )
+
+
+@pytest.mark.parametrize(
+    "min_or_max_value, expected_width",
+    [
+        (None, 10),
+        (1, 1),
+        (123123123123, 20),
+    ],
+)
+def test_numeric_detail_answer_input_class_config(
```

Comment:
I know there is some inconsistencies with the naming of other/detail answers, but this test is testing the `OtherConfig` class, and so the tests should be named as such.

Also, I think for consistency and so the grouping of tests is clearer, the test names should all start with a consistent name e.g. `test_other_config_`.

Reply:
Done